src/components/ui/menu/subMenu.component.tsx

```diff
@@ -79,13 +79,15 @@ class SubMenuComponent extends React.Component<SubMenuProps, ComponentState> {
   private subItemsExpandAnimate = (toValue: number): void => {
     Animated.spring(this.subItemsAnimation, {
       toValue: toValue,
+      useNativeDriver: true,
```

Comment:
breaks the component

Reply:
there is no option to animate width / height with this flag. It leads to a crash